EN/system/str_jp/stringfile_battlevoice.csv

```diff
@@ -577,7 +577,7 @@ battlevoice_chr015_149,That was a fun time. Wanna go again?,
 battlevoice_chr015_150,"We only got to the warm up? Sorry, I don't really hold back.",
 battlevoice_chr015_151,"Oops, was that too much? Sorry, you still in there? I could treat you to some K-BBQ!",
 battlevoice_chr000_683,Hm? Isn't it against the rules to gear up like that in the middle of town?,
-battlevoice_chr000_684,"What am I thinking? Just about the date, of course!",
+battlevoice_chr000_684,"What am I thinking? I'm thinking about our date, silly!",

```

Comment:
This feels like a misread of the line, though I do agree it probably needs changing.
Mario goes: "What the hell are you thinking going on a date?!"
and then Arcueid says "What am I thinking? Obviously I'm thinking about the date!" or something to that effect, completely misunderstanding why Mario would even ask that question to begin with. She sounds legitimately upset that he'd even ask something so obvious.